packages/grafana-data/src/utils/url.ts

```diff
@@ -45,7 +45,7 @@ function toUrlParams(a: any) {
 
   const add = (k: string, v: any) => {
     v = typeof v === 'function' ? v() : v === null ? '' : v === undefined ? '' : v;
-    if (typeof v !== 'boolean') {
+    if (typeof v !== 'boolean' || !v) {
```

Comment:
Sorry for the late response, but I just got back from summer vacation. Can we keep the existing logic and create an inclusive/explicit boolean logic instead of the else on line 50?